ai-eval-service/evaluator.py

```
@@ -1,27 +1,28 @@
 from langfair.metrics.toxicity import ToxicityMetrics
 from langfair.metrics.stereotype import StereotypeMetrics
-from langfair.generator.counterfactual import CounterfactualGenerator
-from langfair.metrics.counterfactual import CounterfactualMetrics
 from dotenv import load_dotenv
 
 load_dotenv()
 
 logger = logging.getLogger(__name__)
 
 
+def _safe_float(value, default=0.0):
```

Ruff (0.14.10)

Missing return type annotation for private function `_safe_float`

(ANN202)